src/lessons/Blog/06.vue

```diff
@@ -140,25 +140,25 @@ const validate = async (result, ipfs) => {
 
   const treeNode = (await ipfs.dag.get(computerNodePrev)).value
   if (treeNode.content === undefined) {
-    return {fail: `The "compputers" blog post should link to the "trees" blog post.`}
+    return {fail: `The "computers" blog post should link to the "trees" blog post.`}
   }
   if (treeNode.content !== 'trees') {
-    return {fail: `The "compputers" blog post should link to the "trees" blog post, but it links to ${treeNode.content}.`}
+    return {fail: `The "computers" blog post should link to the "trees" blog post, but it links to ${treeNode.content}.`}
   }
-  if ('prev' in treeNode) {
+  if (('prev' in treeNode) && (treeNode.prev !== null)) {
     return {fail: 'The "trees" blog post shouldn\'t link to other blog posts.'}
   }
 
   const computerNodePrevCid = computerNodePrev.toBaseEncodedString()
-  if (computerNodePrevCid !== treePostCid) {
+  if ((computerNodePrevCid !== treePostCid) && (computerNodePrevCid !== 'zdpuAoNUinwYTMoTR8Wq7945MKSSpAUNGW1d1wkTHhRcchG3D')) {
```

Comment:
I guess that hash is the treePostCid when `prev: null`. I think it would be clearer if it would also be a variable, something like `treePostCidPrevNull` or so. This way people know where that hash came from.

If you want to have something (un)equal to two items, you could write it also as `[treePostCid, treePostCidPrevNull].includes(computerNodePrevCid)`. I'm unsure if that's clearer or not, I just wanted to mention it.

This whole comment also applies to the hashes below.

Reply:
Both are great suggestions, thanks!